### ISSUE TITLE
NVDA reads three or four data points at a time

### ISSUE THREAD

**Issue by oetseli** [highcharts/highcharts#12151]
#### Expected behaviour
Screen reader reads one data point at a time.

#### Actual behaviour
NVDA reads three to four data points at a time when moving from line to line (down arrow key).

#### Live demo with steps to reproduce
Spotted issue in a project, but you can reproduce it in Highcharts' own sample too: https://www.highcharts.com/samples/highcharts/accessibility/accessible-line

#### Product version
7.2.0

#### Affected browser(s)
Firefox 69.0.2
NVDA 2019.2.1

**Comment by sebastianbochan** [comment]
Hi @oetseli,
Thank you for reporting about the problem.

FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
I spotted this as well, and I think it came with an NVDA update. Not sure if there is much we can do about this, but will investigate further.

**Comment by oetseli** [comment]
Ah, that's good to know. Thanks! I'll try again whenever I notice an NVDA update.

**Comment by oysteinmoseng** [comment]
Updated NVDA again today and the issue still persists. We might consider logging an issue with the NVDA team.

**Comment by oetseli** [comment]
Thank you for the update! It sounds like something they could investigate. I could help by testing older versions of NVDA and confirm that this used to work. I read and tried that the old installers are available if you change the download URL to another version. (maybe there's an archived downloads page somewhere that I'm unaware of)

**Comment by oysteinmoseng** [comment]
Logged issue with the NVDA team, will follow up in that thread and see if we can help them resolve it on their end.

**Comment by ekomisarenko** [comment]
@oysteinmoseng hope everything is well with you. Have you been able to get any update from NVDA on this issue?

**Comment by oysteinmoseng** [comment]
@ekomisarenko No, there seems to be little interest so far from the NVDA team on changing the behavior. There is a [PR discussion](https://github.com/nvaccess/nvda/pull/11906) ongoing on whether or not to disable screen layout by default, which would make this issue less pressing.
